schema/version.json

```diff
@@ -1,4 +1,4 @@
 {
-  "schemaHash": "f0ccc3212331af8a1c75830878d218fd667e5957063831be3b5bfd803b25f0cc",
-  "revision": 39
+  "schemaHash": "bf762e5da559c685e06892ef65fdfc33d1c4bd53d4eb07d4a326d476ac58ae25",
+  "revision": 40
```

Comment:
Should(n't) we release this as a patch on 39? It's not actually adding functionality, it's just more accurately describing current behavior.